.env.example

```diff
@@ -17,3 +17,4 @@ VITE_SUBSCRIPTIONS="https://subscriptions-staging.mutinywallet.com"
 VITE_STORAGE="https://storage-staging.mutinywallet.com"
 VITE_FEEDBACK="https://feedback-staging.mutinywallet.com"
 VITE_SCORER="https://scorer-staging.mutinywallet.com"
+VITE_PRIMAL="https://dev.primal.net/api"
```

Comment:
needs to be swapped to our api and needs to be added to the android build process.